lib/rouletteManager.js

```diff
@@ -96,4 +112,4 @@ RouletteManager.prototype.forceStop = function () {
     return true;
 };
 
-module.exports = RouletteManager;
```

Comment:
Yeah, I tried to re do it, because I did nothing with this line at all; But, it still came up the second time...